### ISSUE TITLE
Fine-tuning LongVILA

### ISSUE THREAD

**Issue by lyluh** [NVlabs/VILA#140]
I am able to fine-tune the original models fine with the /scripts/v1_5/release/3b/3_sft.sh file using my own custom videos, but when I try to train the new LongVILA model I get this warning message and the loss is 0.0 and eval loss is NaN:
<img width="682" alt="Screenshot 2024-09-17 at 4 51 24 PM" src="https://github.com/user-attachments/assets/65428f75-6763-4a1d-a4d4-0e236c3090d8">

Any ideas?

**Comment by liuyijiang1994** [comment]
hi, solved there?

**Comment by badger-lord** [comment]
@lyluh can you share how you created your custom dataset of videos? I'm currently trying this with videos of actions, however the only labels/captions I care about are two words at most. i.e. robotic actions, or the overall idea is asking the finetuned model "did player swing the baseball bat in these 30 frames?" I have short video of baseball player, label of the video "swung bat".  Any help/thoughts?
